workshop/SCHEDULE.md

```diff
@@ -3,17 +3,53 @@ title: Workshop Schedule
 nav_title: Schedule
 ---
 
-<!--See an example from a past remote workshop here: https://github.com/AlexsLemonade/2023-may-training/blob/main/workshop/SCHEDULE.md -->
-<!--See an example from a past in-person workshop here: https://github.com/AlexsLemonade/2023-june-training/blob/main/workshop/SCHEDULE.md -->
+The planned schedule for the June 2024 Data Lab Virtual Training Workshop "Introduction to Single-Cell RNA-Seq" appears below.
 
 
-| Time        | Topic                                          |
-|-------------|------------------------------------------------|
-| **Day 1**   | **Date** <br> <!--uncomment and add link: [Module]()-->                      |
-| 12:00 PM    | Welcome, Introductions and Getting Started     <br>[Welcome Slides (PDF)](../slides/Workshop_Introduction.pdf)|
-| 5:00        | End             |
-| **Day 2**   | **Date**  |
-| **Day 3**   | **Date**  |
-| **Day 4**   | **Date**  |
-| **Day 5**   | **Date**  |
-| 5:00        | Adjourn   |
+*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt).*
+
+
+| Time        | Topic                             | Location |
+|-------------|--------------------------------------------|----------------|
+| **Day 1**   | **2024-06-10** <br> _**Introduction to R and the Tidyverse**_                 |
+| 12:00 PM    | Welcome, Introductions and Getting Started <br> [Workshop introduction slides (PDF)]()  | Zoom: [Main Session](../software-setup/zoom-procedures.md) |
+| 1:00 PM     | [Introduction to base R](../completed-notebooks/01-intro_to_base_R.nb.html) <br> [Introduction to R, RStudio, and RStudio Server slides (PDF)]() | Zoom: [Main Session](../software-setup/zoom-procedures.md)|
```

Comment:
Since the notebook links are broken links at the moment, it probably doesn't make sense to have them here yet. We can add them as they are added to the repo.